src/DependencyInjection/BitBagSyliusMailChimpExtension.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * another great project.
+ * You can find more information about us on https://bitbag.shop and write us
+ * an email on tomasz.grochowski@bitbag.pl.
+ */
+
```

Comment:
We have new text for this IIRC, please ask Krystian about that. And trust me, you don't wont leave your email here 🙉